packages/orbit-components/src/BadgeList/BadgeListItem/index.tsx

```diff
@@ -30,57 +30,43 @@ export const getIconColor = (type: Type) => {
   return type;
 };
 
-export const StyledBadgeListItem = styled.li`
-  ${({ theme }) => css`
-    display: flex;
-    flex-direction: row;
-    width: 100%;
-    & + & {
-      margin-top: ${theme.orbit.spaceXXSmall};
-    }
-  `};
-`;
+export const StyledBadgeListItem = ({ children, ...props }) => (
+  <li className="[&_+_&]:mt-xxs flex w-full flex-row" {...props}>
```

Comment:
The problem is that `StyledBadgeListItem` was receiving `data-test` as prop (or HTML attr), and I can't unpack this (I don't really know if having a variable named "data-test" is possible, probably not in JS). I can change it to `dataTest` and adapt it on `ItineraryBadgeListItem`, but since this component is exported, it would be a breaking change. It's probably not a big deal since, as you said, it is only used there and not exported outside.